src/feature/flow_chart_panel/components/PlayBtn.tsx

```diff
@@ -23,7 +23,12 @@ const PlayBtn = ({ onPlay }: PlayBtnProps) => {
   useKeyboardShortcut("meta", "p", () => onPlay(nodes, edges));
 
   return (
-    <Button size="sm" variant="outline" onClick={handleClick}>
+    <Button
+      size="sm"
+      variant="outline"
+      onClick={handleClick}
+      disabled={!nodes.length}
```

Comment:
I think `nodes.length === 0` is more clear

Reply:
sounds good will change it back